java-frontend/src/main/java/org/sonar/java/se/checks/OptionalGetBeforeIsPresentCheck.java

```diff
@@ -47,15 +47,7 @@
     "NoSuchElementException will be thrown when invoking method %s() without verifying Optional parameter.");
 
   private enum OptionalConstraint implements Constraint {
-    PRESENT, NOT_PRESENT;
-
-    @Override
-    public String valueAsString() {
```

Comment:
This method from is used when computing flows, though the `Constraint` interface. I'm wondering if it's wise to remove it here, as it may have impact when flows are encountering the OptionalConstraints.